src/components/MoviesList/MoviesList.tsx

```diff
@@ -1,6 +1,23 @@
 import React from 'react';
+import { Movie } from '../../types/Movie';
+import { MovieCard } from '../MovieCard';
 import './MoviesList.scss';
 
-export const MoviesList: React.FC = () => (
-  <>Put the list here</>
+type Props = {
+  movies: Movie[]
+};
+
+export const MoviesList: React.FC<Props> = ({ movies = [] }) => (
+  <ul className="movies">
+    {movies.map(movie => (
+      <li key={movie.imdbId} className="card" data-cy="Movie">
+        <MovieCard
+          title={movie.title}
+          description={movie.description}
+          imgUrl={movie.imgUrl}
+          imdbUrl={movie.imdbUrl}
+        />
```

Comment:
Just pass the whole movie and get all the needed props inside MovieCard
![image](https://user-images.githubusercontent.com/96409563/220573030-c14dbe77-fa9f-4dd6-9c5f-7110ab2e2e36.png)